third_party/terraform/tests/resource_access_context_manager_access_level_condition_test.go

```diff
@@ -0,0 +1,161 @@
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAccessContextManagerAccessLevelConditionDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAccessContextManagerAccessLevelCondition_basic(org, "my policy", "level", serviceAccountName),
+				Check:  testAccCheckAccessContextManagerAccessLevelConditionPresent(t, "google_access_context_manager_access_level_condition.access-level-condition", expected),
+			},
+		},
+	})
+}
+
+func testAccCheckAccessContextManagerAccessLevelConditionPresent(t *testing.T, n string, expected map[string]interface{}) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[n]
+		if !ok {
+			return fmt.Errorf("Not found: %s", n)
+		}
+
+		config := googleProviderConfig(t)
+		url, err := replaceVarsForTest(config, rs, "{{AccessContextManagerBasePath}}{{access_level}}")
+		if err != nil {
+			return err
+		}
+
+		al, err := sendRequest(config, "GET", "", url, config.userAgent, nil)
+		if err != nil {
+			return err
+		}
+		conditions := al["basic"].(map[string]interface{})["conditions"].([]interface{})
+		for _, c := range conditions {
+			if reflect.DeepEqual(c, expected) {
+				return nil
+			}
+		}
+		return fmt.Errorf("Did not find condition %+v", expected)
+	}
+}
+
+func testAccCheckAccessContextManagerAccessLevelConditionDestroyProducer(t *testing.T) func(s *terraform.State) error {
+	return func(s *terraform.State) error {
+		for _, rs := range s.RootModule().Resources {
+			if rs.Type != "google_access_context_manager_access_level_condition" {
+				continue
+			}
+
+			config := googleProviderConfig(t)
+
+			url, err := replaceVarsForTest(config, rs, "{{AccessContextManagerBasePath}}{{access_level}}")
+			if err != nil {
+				return err
+			}
+
+			_, err = sendRequest(config, "GET", "", url, config.userAgent, nil)
+			if err == nil {
+				return fmt.Errorf("AccessLevel still exists at %s", url)
+			}
+		}
+
+		return nil
+	}
+}
+
+func testAccAccessContextManagerAccessLevelCondition_basic(org, policyTitle, levelTitleName, saName string) string {
+	return fmt.Sprintf(`
+resource "google_access_context_manager_access_policy" "test-access" {
+  parent = "organizations/%s"
+  title  = "%s"
+}
+
+resource "google_access_context_manager_access_level" "test-access" {
+  parent      = "accessPolicies/${google_access_context_manager_access_policy.test-access.name}"
+  name        = "accessPolicies/${google_access_context_manager_access_policy.test-access.name}/accessLevels/%s"
+  title       = "%s"
+  description = "hello"
+  basic {
+    combining_function = "AND"
+    conditions {
+      device_policy {
+        require_screen_lock = true
+        os_constraints {
+          os_type = "DESKTOP_CHROME_OS"
+        }
+      }
+      regions = [
+  "CH",
+  "IT",
+  "US",
+      ]
+    }
+
+    conditions {
+      ip_subnetworks = ["176.0.4.0/24"]
+    }
+  }
+
+  lifecycle {
+    ignore_changes = [basic.0.conditions]
```

Comment:
This is the parent resource, `access_level`. It has a list of `conditions` specified here, but we add a `condition` later in the test, so ignoring changes to this resource so that there isn't a perma-diff.